app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -42,15 +156,67 @@ const SurveyObservations = () => {
           startIcon={<Icon path={mdiImport} size={1} />}
           variant="outlined"
           color="primary"
-          onClick={() => setOpenImportObservations(true)}>
+          onClick={() => showUploadDialog()}>
           Import
         </Button>
       </Box>
+      {!isLoading && !submissionStatus && (
+        <Box mb={5}>
+          <Typography data-testid="observations-nodata" variant="body2" className={`${classes.infoBox} ${classes.box}`}>
+            No Observation Data.{' '}
+            <Link onClick={() => setOpenImportObservations(true)} className={classes.browseLink}>
+              Click Here to Import
+            </Link>
+          </Typography>
+        </Box>
+      )}
+      {!isLoading && !isValidating && submissionStatus?.status === 'Rejected' && (
+        <div>
+          <Alert severity="error">
+            <AlertTitle>{submissionStatus.fileName}</AlertTitle>
+            Validation Failed
+          </Alert>
+          <Box mb={5} mt={5} display="flex" justifyContent="space-between">
```

Comment:
might want to reduce margin bottom here, may be too much spacing below
```suggestion
          <Box mb={5} mt={5} display="flex" justifyContent="space-between">
```